ouroboros-network/test/Ouroboros/Network/PeerSelection/Test.hs

```diff
@@ -344,11 +345,11 @@ mockPeerSelectionPolicy GovernorMockEnvironment {
       policyGossipOverallTimeout    = 10    -- seconds
     }
 
-interpretPickScript :: (MonadSTM m, Ord peeraddr)
-                    => TVar m PickScript
+interpretPickScript :: (MonadSTMTx stm, Ord peeraddr)
+                    => TVar_ stm PickScript
```

Comment:
Why do we need to change things like this?

Reply:
Neither `TVar` nor `STM` is injective, so if you were to write `TVar m a -> STM m ..` , the monad `m` is entirely ambiguous, and you'd have to introduce a `Proxy`. This seemed cleaner.